ReadMe.md

```diff
@@ -32,10 +32,7 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 * MySQL or PostgreSQL (yet to be decided)
 
 #### Important Note : 
-      The typical MERN Stack involves MongoDB + ExpressJS + ReactJS + NodeJS. But if you see our product, it really needs a 
-      hierarchical database management system not a document oriented. MongoDB doesn't perform well under a lot of situations 
-      so we're planning to use MySQL (or MariaDB) or PostgreSQL [ We have already mentioned that it's yet to be decided 
-      between these two ] . I hope this Note helps people understand why we are not using MongoDB in MERN Stack .
+    Usually MERN Stack is MongoDB , ExpressJS , ReactJS , NodeJS. But for our product, we need a hierarchical database management system not a document oriented that's why we will be using MySQL or PostgreSQL instead of MongoDB.
```

Comment:
Oh, I meant, in the source code view, do the following:

1. Add a line space after the heading.
2. Remove character spaces before each lines.
3. Keep everything what you have written originally in a single line, not multiline.